.gitignore

```diff
@@ -34,6 +34,7 @@ bin
 
 # Ignore files for local deployment
 config/overlays/development/manager_image_patch.yaml
+config/overlays/development/configmap/inferenceservice.yaml
```

Comment:
Why do we want to gitignore this file? isn't this necessary?

Reply:
This file is generated each time we run make deploy-dev. Just like the config/overlays/development/manager_image_patch.yaml we need not commit it.